src/skin/imgloader.cpp

```diff
@@ -25,6 +26,9 @@ QImage* ImgLoader::getImage(const QString& fileName, double scaleFactor) const {
         }
     }
     if (scaleFactor > 1.0) {
+#else
+    if (qApp->devicePixelRatio() > 1.0) {
```

Comment:
Does the scale factor in Preferences still work with this solution? 
I would prefer solution where this is the as because the user might want to override the system scaling setting.